Altalite.py

```diff
@@ -355,11 +348,13 @@ def CRYTERIA():
         else:
             pass
 
+
 def ValuePoints():
     # Todo
     pass
 
-print(start())
+
+print(DisplayTitle())
```

Comment:
To w ogóle można by przenieść do ```start()```. No i print jest zbędny

Reply:
Wrzuciłem commita